eventrecorder/recorder.go

```diff
@@ -189,7 +189,7 @@ func (r *EventRecorder) RecordAggregateEvents(ctx context.Context, events []Aggr
 			  error,
 				protocol
```

Comment:
indenting is off here, mind fixing it before you merge?

Reply:
This was off in the original PR that added it but it's fine in my editor. I've attempted to fix it, but I don't get any diff results when retyping it out, tabbing, etc. Maybe we have to fix it via GitHub editing? 🤔

Reply:
Had to fix it via GitHub. Still no change in my editor when pulling, but at least it now looks good in GitHub. 👍